docs/pages/admin-guides/management/security/proxy-protocol.mdx

```diff
@@ -87,6 +87,20 @@ your network setup:
    from the same IP address from Teleport's point of view, compromising the
    Teleport audit log and IP pinning feature.
 
+1. If you need to connect from an IPv6-only network to an IPv4-only network, you
+   can downgrade the source address by enabling `proxy_protocol_allow_downgrade`.
+   IP pinning is not allowed for downgraded connections due to the possibility of
+   colliding source addresses.
+
+<Admonition type="warning">
```

Comment:
Let's move this admonition to where we explain the `proxy_protocol_allow_downgrade` field in more detail. Otherwise, these instructions are scattered throughout the page.